src/Spice86/UserControls/AddressAutoCompleteBox.axaml.cs

```diff
@@ -0,0 +1,128 @@
+using Avalonia;
+using Avalonia.Controls;
+
+using Spice86.Core.Emulator.CPU;
+using Spice86.Core.Emulator.Memory;
+using Spice86.Shared.Utils;
+
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+
+namespace Spice86.UserControls;
+
+public partial class AddressAutoCompleteBox : UserControl {
```

Comment:
This doesn't actualy autocomplete anything. `AddressAutoConversionBox` might be a better name?

Reply:
File removed.